data/actions/scripts/other/music.lua

```diff
@@ -28,42 +28,47 @@ local instruments = {
 	[3953] = {effect = CONST_ME_SOUND_RED}, -- war drum
 	[3957] = {effect = CONST_ME_SOUND_YELLOW, item = {id = 2681, count = 10}, chance = 80, remove = true}, -- cornucopia
 	[5786] = {effects = {failure = CONST_ME_SOUND_RED, success = CONST_ME_SOUND_YELLOW}, monster = "war wolf", chance = 60, remove = true}, -- wooden whistle
-	[6572] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
-	[6573] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
+	[6572] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transform = {id = 13578}}, -- party trumpet
+	[6573] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transform = {id = 13578}}, -- party trumpet
```

Comment:
Make the simplest code that works. If new features are needed, they are to be added later, not now.